indico/htdocs/sass/partials/_actionboxes.scss

```diff
@@ -84,11 +84,19 @@
     background: $background-color;
     color: $color;
 
-    > .section > .icon {
+    > .section {
+        @include action-box-section-colors($color, $emphasis-color);
+    }
+}
+
+@mixin action-box-section-colors($color, $emphasis-color) {
```

Comment:
Just an idea, since this simply changes text color and I don't think we are using `action-box-section-colors` anywhere else. Couldn't we just simply user `.semantic-text whatever` in case we need to color the text differently.